spec/fixtures/CommonBuildSettings/Project/Objc_watchOS_Native Extension/Assets.xcassets/README__ignoredByTemplate__

```diff
@@ -0,0 +1 @@
+Did you know that git does not support storing empty directories?
```

Comment:
Was this file auto-generated? Aren't `.gitkeep` files the commonly accepted practice for cases like this instead?